lms/djangoapps/badges/admin.py

```diff
@@ -0,0 +1,10 @@
+"""
+Admin registration for Badge Models
+"""
+from django.contrib import admin
+from badges.models import CourseCompleteImageConfiguration, CourseEventBadgesConfiguration, BadgeClass
+from config_models.admin import ConfigurationModelAdmin
+
+admin.site.register(CourseCompleteImageConfiguration)
+admin.site.register(BadgeClass)
+admin.site.register(CourseEventBadgesConfiguration, ConfigurationModelAdmin)
```

Comment:
What does it mean to include two models in one register?  Does it somehow attach ConfigurationModelAdmin to CourseEventBadgesConfiguration?  If so, is the same required for CourseCompleteImageConfiguration above?